package/MDAnalysis/coordinates/DCD.py

```diff
@@ -89,438 +69,214 @@
-        #    struct.unpack("LLiiiiidiPPiiii",self._dcd_C_str)
-        # seems to do the job on Mac OS X 10.6.4 ... but I have no idea why,
-        # given that the C code seems to define them as normal integers
-        desc = [
-            'file_desc', 'header_size', 'natoms', 'nsets', 'setsread', 'istart',
-            'nsavc', 'delta', 'nfixed', 'freeind_ptr', 'fixedcoords_ptr', 'reverse',
-            'charmm', 'first', 'with_unitcell']
-        return dict(zip(desc, struct.unpack("LLiiiiidiPPiiii", self._dcd_C_str)))
 
-    def _reopen(self):
-        self.ts.frame = -1
-        self._reset_dcd_read()
+        .. warning::
+           The DCD format is not well defined. Check your unit cell
+           dimensions carefully, especially when using triclinic boxes.
+           Different software packages implement different conventions and
+           MDAnalysis is currently implementing the newer NAMD/VMD convention
+           and tries to guess the new CHARMM one. Old CHARMM trajectories might
+           give wrong unitcell values. For more details see `Issue 187`_.
 
-    def _read_next_timestep(self, ts=None):
-        """Read the next frame
+        .. versionchanged:: 0.9.0
+           Unitcell is now interpreted in the newer NAMD DCD format as ``[A,
+           gamma, B, beta, alpha, C]`` instead of the old MDAnalysis/CHARMM
+           ordering ``[A, alpha, B, beta, gamma, C]``. We attempt to detect the
+           new CHARMM DCD unitcell format (see `Issue 187`_ for a discussion).
 
-        .. versionchanged 0.11.0::
-           Native frame read into ts._frame, ts.frame naively iterated
+        .. _`X-PLOR DCD format`: http://www.ks.uiuc.edu/Research/vmd/plugins/molfile/dcdplugin.html
+        .. _Issue 187: https://github.com/MDAnalysis/mdanalysis/issues/187
+        .. _DCDplugin: http://www.ks.uiuc.edu/Research/vmd/plugins/doxygen/dcdplugin_8c-source.html#l00947
         """
-        if ts is None:
-            ts = self.ts
-        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, 1)
-        ts.frame += 1
-        return ts
-
-    def _read_frame(self, frame):
-        """Skip to frame and read
+        return self.ts.dimensions
 
-        .. versionchanged:: 0.11.0
-           Native frame read into ts._frame, ts.frame naively set to frame
-        """
-        self._jump_to_frame(frame)
-        ts = self.ts
-        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, 1)
-        ts.frame = frame
-        return ts
+    @property
+    def dt(self):
+        """timestep between frames"""
+        return self.ts.dt
 
-    def timeseries(self, asel=None, start=None, stop=None, step=None, skip=None,
+    def timeseries(self,
+                   asel=None,
+                   start=None,
+                   stop=None,
+                   step=None,
+                   skip=None,
                    format='afc'):
```

Comment:
I think we said so in the Timeseries discussion. Maybe just need an issue for it.